test/test_tasks.py

```diff
@@ -410,6 +410,28 @@ def test_multiple_activities_no_target_urls(self):
     self.assert_equals('complete', resp.status)
     self.assertIsNone(resp.urls_to_activity)
 
+  def test_mentions_only_go_to_posts_and_comments(self):
```

Comment:
wait, that IS what this is testing! (?) activites[0] is marked as a mention; then it tests that the first response (comment) is marked for propagation, and the other two (like and repost) aren't. The name reads backwards to me, I would call it something like `test_only_comments_backfed_through_mentions`